### PR TITLE
Feature/misc feedback fixes

### LINKED ISSUE #1432: Request: Add receipts filter for zip code
Missing Zip Code

image

The current IC search has this field

### LINKED ISSUE #1433: Request: Add primary/general (PGI) election filter for receipts
Add a field which indicates if the contribution is a Primary, General, Special, or Other (when applicable)

### LINKED ISSUE #1434: Request: Make individual contributor name first column on table
I would have the Contributor Name listed FIRST 

image

### LINKED ISSUE #1435: Request: Re-arrange download columns
The CSV file is a MESS.
•	The Contributor should be listed first, not in column L
•	Have the NAME of the committee receiving the contribution be present
•	Too many useless columns:
 	  
image
image
image

The contribution is to the Candidate’s Committee, not the Candidate
     
image
image
image
image
image
image
image
image
image
 
       
•	Redundant columns:
 
image
image
image
image
image
image

•	In Contribution Receipt Date Box:
 
image

It looks there is a place for the time. Needs to be removed
The CSV file should be short, sweet, and to the point.

### LINKED ISSUE #1437: Request: Add "payee" to IE table
Add Payee to the table

### LINKED ISSUE #1439: Question: What are old report types?
What does this mean
image

### LINKED ISSUE #1452: Request: Reorder committees chronologically in candidate profiles
Candidates with more than one committee - in header list in correct order (like lower on the page)

image

### LINKED ISSUE #1460: Request: Change "amount" column to "aggregate amount" in candidate profile -> other spending
Other Spending - Change "amount" column header to "aggregate amount"

### LINKED ISSUE #1478: Request: Include future years in election year filter for senate candidates
Needs future years in election year search for senate candidates

### PR DESCRIPTION
**Pull request by noahmanger**
This clears a bunch of low-hanging fruit from the feedback that @PaulClark2 and @jwchumley shared the other week.
- Adds payee to IE table: Resolves #1437 
- Moves recipient name to first column: Resolves #1434 
- Removes old report types from the filings report type dropdown: resolves #1439 
- Changes "amount" to "aggregate amount" on the other spending tables on candidate and election pages: resolves #1460 
- Fixes the order of committee names in candidate page header so most recent is on top: resolves #1452 
- Adds years up to 2020 in the candidate year selects on the candidate filter pages: resolves #1478 
- Removes district from senate and presidential candidate pages: resolves #1435 
- Adds primary / general election type column (#1433, but doesn't resolve. Still need filter).)

### CODE REVIEW

**Comment by noahmanger** [comment]
Updated PR to remove things that aren't ready. This is now good for review.

File: openfecwebapp/templates/partials/receipts-filter.html [hunk 1]
```diff
@@ -5,7 +5,11 @@
 
 {% import 'macros/filters/states.html' as states %}
 {% import 'macros/filters/date.html' as date %}
+<<<<<<< HEAD
```
Comment (ccostino) [inline openfecwebapp/templates/partials/receipts-filter.html]: D'oh!  Merge conflict code left in place.

**Comment by LindsayYoung** [comment]
Looks good, lots of good little improvements in here

Two thoughts:
- For schedule e, you can do a query by payee name with the API, that would be a good filter to add to the website. 
- Filters on primary and general spending require more indexes. We still have other sort options we want to add so we should prioritize all of those and create benchmarking to figure out how many we should/can add.

File: openfecwebapp/templates/partials/receipts-filter.html (hunk 1 above)
Comment (LindsayYoung) [inline openfecwebapp/templates/partials/receipts-filter.html]: I don't have this file locally and I don' see it when I look at the diffs.

Comment (LindsayYoung) [inline openfecwebapp/templates/partials/receipts-filter.html]: Because it is fixed! Thanks

**Comment by noahmanger** [comment]
Thanks! Good to hold off on the primary / general filter. I just added payee filter.

**Comment by LindsayYoung** [comment]
AWESOME

**Pull request merged**